README.md

```diff
@@ -160,6 +160,23 @@ const menuTemplate = new MenuTemplate<MyContext>((ctx) => {
 });
 ```
 
+### Can I use entities or `grammyjs/parse-mode` in the message body?
+
+Yes, you can pass the option `entities`.
+
+This will be resolved automatically to`caption_entities` if you send a `media` message.
+
+```ts
+const menu = new MenuTemplate<MyContext>(async () => {
+	const message = fmt`${bold(underline('Hello world!'))}`
+
```

Comment:
Removing the empty line seems more concise to me.

```suggestion
```

Reply:
Thanks a lot for your feedback, I updated the documentation